macros/query_comment.sql

```diff
@@ -53,36 +53,42 @@
                 materialized=node.config.materialized,
             ) -%}
         {%- endif -%}
+
+        {%- if node.raw_code is not none -%}
+            {%- do comment_dict.update({
+                "dbt_raw_code_hash": local_md5(node.raw_code)
+            }) -%}
+        {%- endif -%}
```

Comment:
@bmoore813 just curious, does `raw_code` refer to the raw SQL with the jinja references + model config block at the top?

Reply:
I love the question and happy to test further but it was my understanding that depending on the node in jinja it could be an option of [these](https://schemas.getdbt.com/dbt/manifest/v11/index.html#nodes) so if it exists then it pulls from the graph and applies what the `raw_code` is....How do you think I could better test to ensure this is the raw dbt code that is in question

Reply:
but yes I would assume this is what the code looks like on the repo

Reply:
I just looked at what the `raw_code` looks like in the manifest.json and it is indeed exactly as I just suggested.